vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -192,9 +196,12 @@ class RoomDetailFragment :
 
     private lateinit var actionViewModel: ActionsHandler
     private lateinit var layoutManager: LinearLayoutManager
+    private lateinit var attachmentsHelper: AttachmentsHelper
+    private lateinit var keyboardStateUtils: KeyboardStateUtils
```

Comment:
Let the activity manages KeyboardStateUtils?

Reply:
I prefer not!

Reply:
why?